src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {

```

Comment:
Did you run your code? 
The result of method is: Bulldozer: Bulldozer start working

Why do you need Bulldozer twice?